src/Dropdown.elm

```diff
@@ -414,7 +432,9 @@ triggerView config state =
                         )
                         { onChange = config.dropdownMsg << OnFilterTyped
                         , text = state.filterText
-                        , placeholder = Just <| Input.placeholder [] (text config.filterPlaceholder)
+                        , placeholder =
+                            config.filterPlaceholder
```

Comment:
This new version this could result in a Just Nothing, isnt?

Reply:
what is the intended behavior? As I understand it, if I don't want a placeholder I put a Nothing in filterPlaceholder and the rendered html should no put a placeholder. If I put something (f.e. placeholder = Just "start typing to filter") then the rendered html should show it. That's why I suggested to just put Nothing when the filterPlaceholder is Nothing, instead of Just (Input.placeholder [] none) when the filterPlaceholder is Nothing.
I think I am not explaining myself correctly :(

Reply:
How...? Maybe i dont think right about it. If . config.filterPlaceholder is Nothing then Maybe.map maps the Nothing right so returns Nothing...

Reply:
ahh, nevermind, I was looking at the old code. Ignore me, I am drunk!

Reply:
it is ok now, sorry for the confusion :D